pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,137 @@
+import os
+import re
+import subprocess
+
+import click
+import koji
+import yaml
+from errata_tool import Erratum
+from pyartcd import constants, util
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+
+class OperatorSDKPipeline:
+    def __init__(self, runtime: Runtime, group: str, assembly: str, nvr: str, prerelease: bool, updatelatest: bool) -> None:
+        self.runtime = runtime
+        self._logger = runtime.logger
+        self.version = group.split("-")[1]
+        self.assembly = assembly
+        self.prerelease = prerelease
+        self.nvr = nvr
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+        self.group = group
+        self.extra_ad_id = ""
+        self.parent_jira_key = ""
+        self._jira_client = runtime.new_jira_client()
+
+    async def run(self):
+        if self.assembly:
+            group_config = await util.load_group_config(self.group, self.assembly, env=os.environ.copy())
+            self.extra_ad_id = group_config.get("advisories", {}).get("extras", 0)
+            self.parent_jira_key = group_config.get("release_jira")
+            advisory = Erratum(errata_id=self.extra_ad_id)
+            self._logger.info("Check advisory status ...")
+            if advisory.errata_state in ["QE", "NEW_FILES"]:
+                self._logger.info("Advisory status not in REL_PREP yet ...")
+                return
+            if advisory.errata_state == "SHIPPED_LIVE":
+                self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
+                self._update_jira(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
+            self._logger.info("Advisory status already in post REL_PREP, update subtask 7 ...")
+            self._update_jira(self.parent_jira_key, 6, "Advisory status already in REL_PREP")
+
+            et_builds = advisory.errata_builds
+            sdk_build = [i for i in et_builds[f'OSE-{self.version}-RHEL-8'] if re.search("openshift-enterprise-operator-sdk-container*", i)]
+            if not sdk_build:
+                self._logger.info("No SDK build to ship, update subtask 8 then close ...")
+                self._update_jira(self.parent_jira_key, 7, f"No SDK build to ship, operator_sdk_sync job: {os.environ.get('BUILD_URL')}")
```

Comment:
same here.

Reply:
Also please use the function in Runtime to get the build URL so we will have a single place to change after moving to Tekton.